config/adminlte.php

```diff
@@ -333,6 +333,11 @@
             'url' => '/students',
             'icon' => 'fa fa-users',
         ],
+        [
+            'text' => 'Categorias',
```

Comment:
Nada más aquí, con acento en la i
```suggestion
            'text' => 'Categorías',
```